README.md

```diff
@@ -3,6 +3,7 @@
 # Knavigator
 
 ![Build Status](https://github.com/NVIDIA/knavigator/actions/workflows/go.yml/badge.svg)
+[![codecov](https://codecov.io/github/shinae-woo/knavigator/graph/badge.svg?token=L6HOTRHFD4)](https://codecov.io/github/shinae-woo/knavigator)
```

Comment:
Does it have to be "shinae-woo" in the URL?

Reply:
It will be nvidia :)